src/app/_components/ArticleEditor.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { EditorState } from 'prosemirror-state';
+import { EditorView } from 'prosemirror-view';
+import { DOMParser as ProseMirrorDOMParser } from 'prosemirror-model';
+import { schema } from 'prosemirror-schema-basic';
+import { exampleSetup } from 'prosemirror-example-setup';
+import { defaultMarkdownParser, defaultMarkdownSerializer } from 'prosemirror-markdown';
+import 'prosemirror-view/style/prosemirror.css';
+import 'prosemirror-menu/style/menu.css';
+import 'prosemirror-example-setup/style/style.css';
+
+const editorStyles = `
+  .ProseMirror h1 {
+    font-size: 24px;
+    font-weight: bold;
+    margin-top: 20px;
+    margin-bottom: 10px;
+  }
```

Comment:
Instead of doing this, please use a [css module](https://nextjs.org/docs/pages/building-your-application/styling/css-modules) as specified by nextjs.